dist/module/funding.js

```diff
@@ -55,4 +56,4 @@ export var FUNDING_PRODUCTS = {
   PAY_IN_4: "payIn4",
   PAYLATER: "paylater",
   CREDIT: "credit"
-};
\ No newline at end of file
+};
```

Comment:
Please remove the "dist" folder changes in this PR. Our publishing process takes care of updating the dist folder.

Reply:
Removed the "dist" folder